### DISCUSSION TITLE
I have created a Japanese translation file.

### DISCUSSION THREAD

**Discussion by KKbion** [Klocman/Bulk-Crap-Uninstaller#496] [General]
**I have created a Japanese translation file.**

[20230527_BCU_jp_Added_Localisation_Pack_5.6.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/11577068/20230527_BCU_jp_Added_Localisation_Pack_5.6.zip)

ーーーー
Hi, I am a user of BCU for 3 years now.
I could not find a localization pack in Japanese, so I created one myself. I created a natural translation of all the items that can be translated.

The parts of the text that start with "&" and look like commands, such as Uninstall lists|*.bcul, are translated as "アンインストール lists|*.bcul", etc.

Besides, can I refer to both "Silent" Uninstall and "Quiet" Uninstaller as Silent Uninstaller? Or should they be listed separately as they currently are? I have never used that feature, so I don't know how to handle the proper translation.

For the time being, I think this will make it easier for Japanese users to use BCU. I would be happy if we can adapt it in the next update, etc.

**Also, my name as a contributor is ”LLY kindness people”.** 

Thank you　く(^_^)

**Comment by Klocman** [comment]
Thank you! I always wanted to add a Japanese translation but lacked the skill. I'm certain it will help a lot of people. I'll merge the translation in the next release.

In case of the "アンインストール lists|.bcul", the whole part on the left of | can be translated (including "lists"), while the part on the right of | must stay untouched (.bcul). In case of "A|.abc|B|.bcd|C|.cde" A, B and C can be translated while everything else should stay untouched.

"Silent" Uninstall refers to the process of uninstalling with no user input, while "Quiet" Uninstaller refers to one particular uninstaller that does not require user input. That being said, "Silent" and "Quiet" can be used interchangeably. You can pick whatever is easier to translate as long as it's consistent across the translation.

**Reply by KKbion** [reply]
Thank you for mentioning the description method. I will try to modify it sequentially in the future when my hands are free. Thank you very much.　^^

**Reply by KKbion** [reply]
Hi, I updated the Japanese translation file today.

[2023jun19_JP_BCULocalisation_Pack.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/11786121/2023jun19_JP_BCULocalisation_Pack.zip)

As you pointed out the other day, I have changed the notation as "アンインストール lists|.bcul" -> "アンインストールリスト|.bcul".

Also, since the contents of Silent Uninstall and Quiet Uninstall seemed to be almost the same, I have unified the notation to "Silent Uninstall" for both.

**Reply by KKbion** [reply]
@Klocman 
Please check the updated Japanese file here.
